PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -12,8 +12,10 @@ import Purchases
 class PurchasesTests: XCTestCase {
 
     override func setUp() {
-        deviceCache = MockDeviceCache()
         userDefaults = UserDefaults(suiteName: "TestDefaults")
+        userDefaults.dictionaryRepresentation().keys.forEach { userDefaults.removeObject(forKey: $0) }
```

Comment:
UserDefaults wasn't clean.

Reply:
that must have been a fun one to figure out